docs/source/io_formats/data_wmp.rst

```diff
@@ -0,0 +1,92 @@
+.. _io_data_wmp:
+
+==========================================
+The Windowed Multipole Library Format v0.2
+==========================================
+
+**/nuclide/**
+    - **broaden_poly** (*int[]*)
+        If 1, Doppler broaden curve fit for window with corresponding index.
+        If 0, do not.
+    - **curvefit** (*double[][][]*)
+        Curve fit coefficients. Indexed by (reaction type, coefficient index,
+        window index).
+    - **data** (*complex[][]*)
+        Complex poles and residues.  Each pole has a corresponding set of
+        residues.  For example, the `i`th pole and corresponding residues are
+        stored as `data[:,i] = [pole, residue_1, residue_2, ...]`.  The 
+        residues are in the order: total, competitive if present, absorption,
+        fission. Complex numbers are stored by forming a type with `"r"` and
+        `"i"` identifiers, similar to how `h5py` does it.
```

Comment:
This description renders weirdly. Inline literals should use double backticks. Inline math should use `` :math:`...` ``. The h5py reference could be a link to h5py documentation.